### DISCUSSION TITLE
UM vn13.4

### DISCUSSION THREAD

**Discussion by jennyhickson** [MetOffice/simulation-systems#90] [UM]
Dear all,

I'm happy to announce that UM vn13.4 has now been released. Many thanks to all developers and reviewers for their efforts and contributions.

 For Met Office users - prebuilds have been set to compile and will appear over the next few hours. Please build with "-S PREBUILDS=false" if you want to use rose-stem before they are ready.

Documentation and Release Notes will follow shortly.

 Many thanks,

Jenny, on behalf of the SSD Team

**Comment by richardgilham** [comment]
The UM Docs have now been released, and for Met Office users the code- and STASH-browsers have been updated.

On the LFRic side, revisions for UM, JULES etc dependencies have been reset and the um13.4 revision keyword will be up imminently.
